src/ResourceManager/Reservations/ChangeLog.md

```diff
@@ -20,6 +20,14 @@
 -->
 ## Current Release
 
+## Version 0.1.4
```

Comment:
todo: update this appropriately and find out if any other places need version change

Reply:
move this to 'Current Release' section